components/eventing-controller/pkg/backend/mocks/JetStreamContext.go

```diff
@@ -0,0 +1,1061 @@
+// Code generated by mockery v2.14.0. DO NOT EDIT.
```

Comment:
I really don't see a benefit for it right now for different reasons:
- I don't see a problem in exposing the autogenerated mocks
- having all the mocks in one file could cause difficulties
- we already have a lot of files in the source code package
- we already had other mocks in the same mocks package

Reply:
With the change I proposed we would still have several files. For `JetStreamContext.go` it would be `JetStreamContext_test.go`. The benefit is that the mock can never be used in production code since it would be only available in a `test_.go` file.

Reply:
I honestly don't see a big problem of having the mocks in our "production" code.

Reply:
Okay, let's leave it as it is. I will have face the same question in my [PR](https://github.com/kyma-project/kyma/pull/15720/files?diff=unified&w=0) and try to follow the approach. I will see if it causes any problems and report back to the team 😃